src/main/java/hudson/plugins/git/util/DefaultBuildChooser.java

```diff
@@ -120,7 +120,7 @@ public DefaultBuildChooser() {
                 verbose(listener, "{0} seems to be a non-branch reference (tag?)");
             }
         }
-        
```

Comment:
Sure thing. I guess my editor was set to strip trailing whitespace from empty lines in java src. I don't write java that often so I didn't notice it.